mx.sulong/mx_sulong.py

```diff
@@ -509,6 +504,31 @@ def runTests(args=None):
         command = testCases[testSuiteName]
         command(vmArgs)
 
+def checkCode(args=None):
```

Comment:
One (optional) fancy thing to do would be to use a table like in `mx su-tests` to let the user select a subset of static analysis checks to be executed.